packages/artillery/lib/platform/aws/aws-cloudwatch.js

```diff
@@ -47,16 +47,36 @@ function setCloudwatchRetention(
         );
         clearInterval(interval);
       } catch (error) {
-        if (error & (error.code != 'ResourceNotFoundException')) {
-          console.log('WARNING: Unexpected error setting retention policy:');
-          console.log(error);
+        const resumeTestMessage =
+          'The test will resume without setting the retention policy.';
+        if (error?.code == 'AccessDeniedException') {
+          console.log(`\n${error.message}`);
+          console.log(
+            '\nWARNING: Missing logs:PutRetentionPolicy permission to set Cloudwatch retention policy. Please ensure the IAM role has the necessary permissions:\nhttps://www.artillery.io/docs/load-testing-at-scale/aws-fargate#iam-permissions'
```

Comment:
"Cloudwatch" -> "CloudWatch"

And we can make that link a docs.art link so that it's shorter